rskj-core/build.gradle

```diff
@@ -416,3 +533,8 @@ static def amendPathIfNeeded(details) {
         details.path = newPath
     }
 }
+
+tasks.named('check').configure {
+    dependsOn 'checkstyleMain'
+    dependsOn 'checkstyleTest'
```

Comment:
question:

What does this method? It's creating a `check` task? Will this check also run the `checkstyleIntegrationTest` and  `checkstyleJmh`? Because if not, I would suggest to add them here as well. If we are adding a global check, it should check everything.

Reply:
Addressed! I’ve just added `checkstyleJmh` and `checkstyleIntegrationTest` to our `build.gradle` file. Now, when you run the `check` task, it will automatically trigger style checks for `checkstyleMain`, `checkstyleTest`, `checkstyleJmh`, and `checkstyleIntegrationTest`. This means you no longer need to manually run each check individually. Instead, running the check task will execute all these style checks in one go, making the process more efficient